charts/ssi-dim-wallet-stub/Chart.yaml

```diff
@@ -0,0 +1,35 @@
+###############################################################
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################
+
+apiVersion: v2
+name: ssi-dim-wallet-stub
+description: |
+  A Helm chart to deploy SSI DIM wallet stub in kubernetes cluster
+type: application
+version: 0.1.0
+appVersion: "0.0.1"
+home: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/tree/main/charts/ssi-dim-wallet-stub
+sources:
+  - https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/tree/main/charts/ssi-dim-wallet-stub
+dependencies:
+  - name: keycloak
```

Comment:
would it be possible to use the helm chart for centralidp directly https://github.com/eclipse-tractusx/portal-iam/tree/main/charts/centralidp?

Reply:
This application needs keycloak when it pushes did document to portal backend after a wallet is created, and API to push data in portal backend, we need to create access_token using from keycloak.

I avoided centralidp for further complexity. If it is necessary, then we can add it.

Reply:
I don't quite understand why adding centralidp Keycloak would add complexity, I think it would make things more easy to understand, for instance you even refer [here](https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/2/files#diff-8c14e17c4135ba3fa0819e6018b1fdba9edd22987bdec0e83f47fd25034f7c2fR130) to the CX-Central realm which is only available in centralidp Keycloak...
If you'd like to leave it with the generic Bitnami Keycloak, could you please add a comment here stating that the the equivalent Keycloak Instance in the Eclipse Tractus-X ecosystem is the centralidp Keycloak https://github.com/eclipse-tractusx/portal-iam/tree/v3.0.0/charts/centralidp

Reply:
centralidp added in dependencies : https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/2/commits/42d0254d1400cc02f2974d41e5d6630c8e58ed0e